flow-typed/platform.js

```diff
@@ -7,44 +7,51 @@ declare type MatchMedia = {
 	isAtLargeUp: boolean,
 };
 
+declare type AppContext = {
+	apiUrl: string, // base API URL for queries
```

Comment:
Would it be useful to add `location` to AppContext as well?

Reply:
it might help reduce some argument passing in the server render code, but it would be misleading in the client-side application because it wouldn't be clear that `state.config.location` refers only to the server-rendered page, even when the user navigates elsewhere. I think it's probably better to leave it out